content/blog/2023/08/18/2023-08-18-summer-internship-in-jenkins-security.adoc

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Summer Internship in Jenkins security"
+tags:
+- plugins
+- security
+- jenkins
+authors:
+- andreachiera
+- kguerroudj
+- wadeck
+opengraph:
+  image: /images/post-images/2023/08/18/2023-08-18-summer-internship-in-jenkins-security.png
+---
+
+=== Context
+
+Jenkins is an open-source CI/CD solution that is extensible with a wide number of plugins that can be installed using the Jenkins plugin distribution repository or via manual installation.
```

Comment:
```suggestion
Jenkins is an open-source CI/CD solution that is extensible with a wide range of plugins that can be installed using the Jenkins plugin distribution repository or via manual installation.
```
small word suggestion for syntax, another option would be "high number"

Reply:
thank you so much  for your suggestions. I really appreciated them 💪

Reply:
thank you so much for your suggestions. I really appreciated them 💪